js/client-mainmenu.js

```diff
@@ -168,8 +168,13 @@
 
 		addPM: function (name, message, target) {
 			var userid = toUserid(name);
-			if (app.ignore[userid] && name.substr(0, 1) in {' ': 1, '+': 1, '!': 1, '✖': 1, '‽': 1}) return;
-
+			if (app.ignore[userid] && name.substr(0, 1) in {' ': 1, '+': 1, '!': 1, '✖': 1, '‽': 1}) {
+				if (!app.ignoreNotified) {
+					message = '/nonotify A message from ' + BattleLog.escapeHTML(name) + ' was ignored.';
```

Comment:
This doesn't need an HTML escape, right?

Reply:
Turns out it does, `/nonotify` is in fact HTML. What a bad name for it. :/

Reply:
Shouldn't it? it gets appended into the PM.

Reply:
Oh, didn't see your comment. MB.